components/automate-ui/wallaby.js

```diff
@@ -1,93 +1,157 @@
-// This "config" file comes from https://github.com/wallabyjs/ngCliWebpackSample/ .
-// There are only a couple tweaks needed for Automate-UI, noted inline.
+// This "config" file comes from:
+// URL: https://github.com/wallabyjs/ngCliWebpackSample/
+// Commit: 59ed9ba96ca00b21d1f637b593afa7e245ccae3b
```

Comment:
This is actually a vendored file. The good folks from Wallaby brought this file up-to-date very quickly for Angular 8 upon my request.